data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+				accept
+	npc
+		government " Drak "
+		system Nenia
+		personality staying heroic nemesis uninterested
+		ship "Archon" "Lifted Lorax"
+	on stopover
+		conversation
+			`As you approach Slylandro you watch the Archon like a hawk, but this time it doesn't attempt to drive you off with tridents of white energy. Cautiously you make your way to Slylandro and begin your descent.`
+			`	As <ship> sinks through the cloud layers, you notice the void sprites seem to be tailing you. A few drop past you with startling speed, only to reappear from below as you approach the designated altitude. At the last minute you remember Plume's request to record them, and hit the switch to activate the recorder for the exterior cameras.`
+			`	Setting the ship on auto-pilot, you head back and suit-up in the bulky pressure suit once again. Readying the eggs in the cargo hold, you equalize pressure and open the cargo bay doors. As before, the eggs are already floating as you open the cocoons and gently carry them to the door.`	
+			`	Almost as soon as you release the first egg, a void sprite is there, brushing against the ship to scoop the egg away into the swirling mists. By the time you return with the next one, another sprite is already waiting at the door, undulating in the eerie light. By the time the last egg is released you can see dozens of the strange creatures swarming around the collection of eggs. It occurs to you that you very few humans have ever seen anything like this.`
+			`	After a moment of enjoying the view, you shut the door as gently as you can, purge the ship with fresh air, and begin the journey back to Aventine.`
+	on complete
+		conversation
+			`Back on Aventine, you share the recording of the void sprite's behavior with Plume. "These Sprites demonstrate considerably more communication than we originally thought. And obviously they must have communicated what you did on Nasqueron. Interesting...." Plume's excited staccato exclamations trail off as he begins to make notes on a datapad he pulls out. You let him go off on his rambling ideas, shaking your head in amusement.`
+		"reputation:  Drak " += 1
+
+		
+mission "Remnant: Learning Sign 1"
+	name "Learning Remnant Sign"
+	description `Visit each of the Remnant worlds to find ways to learn their sign language.`
+	source "Caelian"
+	stopover "Aventine"
+	destination "Viminal"
+	to offer
+		has "license: Remnant"
+		random < 75
+	on offer
+		conversation
+			`As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you note that it doesn't seem any less busy than usual, just that fewer people are paying any attention to you. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating to realize that there are hundreds of conversations going on around you, and you can't understand a word of it.`
```

Comment:
"As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you note that it doesn't seem any less busy than usual, just that fewer people are paying any attention to you. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating to realize that there are hundreds of conversations going on around you, and you can't understand a word of it."
To;
"As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you notice that it doesn't seem any less busy than usual, just that _only a handful of people are paying any attention to you_. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating _, and eerie,_ to know that there are hundreds of conversations going on around you, _yet_ you can't understand a _single_ word of it."

Reply:
note the spacings, since I'm trying the italics to make the changes easily findable

Reply:
OK. I'm not copy/pasting, so that should be fine. I appreciate the italics making things more visible. 

Your changes do help make the intention of the paragraph easier to understand. 👍